server/terraform/README.md

```diff
@@ -257,6 +260,27 @@ Production servers must be configured exactly as described here. Changes should
 be made through code and merged before being manually applied. Exceptions to this are
 defined in https://github.com/WorldHealthOrganization/app/blob/master/server/terraform/README.md#Exceptions.
 
+### Create Firestore Database
+
+> **_NOTE_**: ONLY FOR V2 (FIRESTORE NATIVE) BACKENDS
+
+Create the Firestore database by...
+
+- Going to the [Firebase Console](console.firebase.google.com)
+- Selecting your newly created project.
+- Clicking the `Cloud Firestore` item in the menu on the left.
+- Clicking the `Create database` button. Leave the settings on default and click `Next`, then `Enable`.
```

Comment:
@rjhuijsman Seems strange that this can't be automatically done by Terraform?

Reply:
I 100% agree that it's strange! I've spent quite some time trying to make it happen automatically! I wonder if it's a regression. I've reported a bug using Firebase's [form for that](https://firebase.google.com/support/troubleshooter/report/bugs).